content/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch.md

```diff
@@ -42,7 +42,7 @@ From the local clone of the repository on a computer, run the following commands
 ```shell
 $ git branch -m OLD-BRANCH-NAME NEW-BRANCH-NAME
 $ git fetch origin
-$ git branch -u origin/NEW-BRANCH-NAME NEW-BRANCH-NAME
+$ git branch -u origin/OLD-BRANCH-NAME NEW-BRANCH-NAME
```

Comment:
Thanks for reporting this! I think that this new command will set the upstream tracking branch for your local branch `NEW-BRANCH-NAME` to the remote branch `origin/OLD-BRANCH-NAME`. However, I think pushing your new branch will resolve the error that you are seeing and allow you to set the upstream tracking branch to be your new branch:

```suggestion
$ git push origin NEW-BRANCH-NAME
$ git branch -u origin/NEW-BRANCH-NAME NEW-BRANCH-NAME
```